Workbooks/Resource Insights/AzureDataExplorerCluster/ResourceInsights.workbook

```diff
@@ -201,40 +207,369 @@
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-CacheUtilization Timeline",
+                    "formatter": 9,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-CPU",
+                    "formatter": 8,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 1,
+                      "options": {
+                        "style": "decimal"
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-CPU Timeline",
+                    "formatter": 9,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-IngestionUtilization",
+                    "formatter": 8,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 1,
+                      "options": {
+                        "style": "decimal"
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-IngestionUtilization Timeline",
+                    "formatter": 9,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-InstanceCount",
+                    "formatter": 4,
+                    "formatOptions": {},
+                    "numberFormat": {
+                      "unit": 0,
+                      "options": {
+                        "style": "decimal"
+                      }
+                    }
+                  },
+                  {
+                    "columnMatch": "microsoft.kusto/clusters-Cluster health-InstanceCount Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "Extents (Total)",
+                    "formatter": 4,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "Extents (Total) Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "Throttled Commands (Total)",
+                    "formatter": 8,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "Throttled Commands (Total) Timeline",
+                    "formatter": 5,
+                    "formatOptions": {}
+                  },
+                  {
+                    "columnMatch": "microsoft.cache/redis--usedmemorypercentage",
```

Comment:
I think these column settings were copied over from previosu template, we should clean this up to reduce size of workbook